package.json

```diff
@@ -77,7 +77,8 @@
   "scripts": {
     "start": "node bin server",
     "prepublishOnly": "npm ls && npm test",
-    "test": "set -e; for t in test/*.js; do echo TEST $t; node $t; done && ./test/deps.sh"
+    "test": "set -e; npm ls --package-lock && for t in test/*.js; do echo TEST $t; node $t; done && ./test/deps.sh",
+    "reinstall": "rm -rf node_modules && npm install --package-lock && npm shrinkwrap"
```

Comment:
isn't that how you get npm to do a clean install and update all deps? oh maybe it should `rm npm-shrinkwrap.json` too? it's to get npm to behave in a predictable way.

npm shrinkwrap turns `package-lock.json` into `npm-shrinkwrap.json`, my understanding is this is how you create a shrinkwrap.

Reply:
on my setup I need `--package-lock` because I have package-lock disabled by default.
I just see too many weird npm errors that I don't really understand to trust that npm really works (like sometimes I npm install something then immediately get an error that module is missing...)
so when I want to be sure I `rm -rf node_modules`